src/math/ceil.js

```diff
@@ -0,0 +1,35 @@
+const ceil = (number, precision) => {
+    let numberString = number.toString();
+    let decimalIndex = numberString.indexOf('.');
+    let beforeDecimal = numberString.substring(0, decimalIndex);
+    let afterDecimal = numberString.substring(decimalIndex, numberString.length);
+  if (Number.isInteger(number)){
+    return number;
+  } else if (Number.isInteger(number) === false) {
+      if (precision === undefined || precision  === 0) {
+        for(let i = 0; i < afterDecimal.length; i++){
+          if(numberString[i] > 0){
+            let roundUp = parseInt(beforeDecimal) + 1;
+            return roundUp;
+          }
+        }
+      } else if (precision > 0) {
```

Comment:
This inner if..else can be inverted for cleaner logic and code. 
Rather than checking `if (precision === undefined || precision  === 0)` and then checking `else if (precision > 0) ` you can do one if without an else if like: 
```
if (precision > 0) {
  // do things when the precision is greater than 0
} else {
  // do things when precision is undefined or if its === 0
}
```